example/CMakeLists.txt

```diff
@@ -1,41 +1,44 @@
-#------------------------------------------------------------------------------
-# © 2021-2025. Triad National Security, LLC. All rights reserved.  This
-# program was produced under U.S. Government contract 89233218CNA000001
-# for Los Alamos National Laboratory (LANL), which is operated by Triad
-# National Security, LLC for the U.S.  Department of Energy/National
-# Nuclear Security Administration. All rights in the program are
-# reserved by Triad National Security, LLC, and the U.S. Department of
-# Energy/National Nuclear Security Administration. The Government is
-# granted for itself and others acting on its behalf a nonexclusive,
-# paid-up, irrevocable worldwide license in this material to reproduce,
-# prepare derivative works, distribute copies to the public, perform
+# ------------------------------------------------------------------------------
+# © 2021-2025. Triad National Security, LLC. All rights reserved.  This program
+# was produced under U.S. Government contract 89233218CNA000001 for Los Alamos
+# National Laboratory (LANL), which is operated by Triad National Security, LLC
+# for the U.S.  Department of Energy/National Nuclear Security Administration.
+# All rights in the program are reserved by Triad National Security, LLC, and
+# the U.S. Department of Energy/National Nuclear Security Administration. The
+# Government is granted for itself and others acting on its behalf a
+# nonexclusive, paid-up, irrevocable worldwide license in this material to
+# reproduce, prepare derivative works, distribute copies to the public, perform
 # publicly and display publicly, and to permit others to do so.
-#------------------------------------------------------------------------------
+# ------------------------------------------------------------------------------
 
-add_executable(get_sound_speed_press
-  get_sound_speed_press.cpp)
-target_link_libraries(get_sound_speed_press PRIVATE
-  singularity-eos::singularity-eos)
+add_executable(get_sound_speed_press get_sound_speed_press.cpp)
+target_link_libraries(get_sound_speed_press
+                      PRIVATE singularity-eos::singularity-eos)
 
-if (SINGULARITY_USE_SPINER_WITH_HDF5)
+add_executable(Benchmark Benchmark.cpp)
+target_link_libraries(Benchmark PRIVATE singularity-eos::singularity-eos
+                                        stdc++fs)
+
+if(SINGULARITY_USE_KOKKOS)
+  add_executable(Benchmark_kokkos Benchmark_kokkos.cpp)
+  target_link_libraries(Benchmark_kokkos
+                        PRIVATE singularity-eos::singularity-eos stdc++fs)
+endif()
```

Comment:
It seems to me that these should be wrapped in the `if (SINGULARITY_USE_SPINER_WITH_HDF5)` check since they both use spiner, right?

Reply:
In fact, I'd use `if (SINGULARITY_USE_EOSPAC AND SINGULARITY_USE_SPINER_WITH_HDF5)` to make sure you solve the CI issue where it can't find EOSPAC:
```
In file included from /home/runner/work/singularity-eos/singularity-eos/example/Benchmark.cpp:47:
/home/runner/work/singularity-eos/singularity-eos/eospac-wrapper/eospac_wrapper.hpp:24:10: fatal error: 'eos_Interface.h' file not found
   24 | #include <eos_Interface.h> // eospac API
      |          ^~~~~~~~~~~~~~~~~
```

Reply:
You _could_ instead use `#ifdef` preprocessor macros in the file itself, but I think it's better to just not compile the example if the appropriate conditions aren't met.

Reply:
That works!